platform.md

```diff
@@ -735,43 +735,38 @@ Usage:
 #### `launcher`
 Usage:
 ```
-/cnb/process/<process-type> [<arg>...]
+/cnb/process/<process-type> [<args>...]
 # OR
-/cnb/lifecycle/launcher [--] [<cmd> <arg>...]
+/cnb/lifecycle/launcher [<cmd> <args>...]
 ```
 ##### Inputs
 | Input                              | Environment Variable | Default Value | Description                                               |
 |------------------------------------|----------------------|---------------|-----------------------------------------------------------|
 | `<app>`                            | `CNB_APP_DIR`        | `/workspace`  | Path to application directory                             |
 | `<layers>`                         | `CNB_LAYERS_DIR`     | `/layers`     | Path to layer directory                                   |
 | `<process-type>`                   |                      |               | `type` of process to launch                               |
-| `<direct>`                         |                      |               | Process execution strategy                                |
 | `<cmd>`                            |                      |               | Command to execute                                        |
 | `<args>`                           |                      |               | Arguments to command                                      |
 | `<layers>/config/metadata.toml`    |                      |               | Build metadata (see [`metadata.toml`](#metadatatoml-toml) |
 | `<layers>/<buildpack-id>/<layer>/` |                      |               | Launch Layers                                             |
 
-A command (`<cmd>`), arguments to that command (`<args>`), a working directory (`<working-dir>`), and an execution strategy (`<direct>`) comprise a process definition. Processes MAY be buildpack-defined or user-defined.
+A command (`<cmd>`), arguments to that command (`<args>`), and a working directory (`<working-dir>`) comprise a process definition. Processes MAY be buildpack-defined or user-defined.
 
 The launcher:
-- MUST derive the values of `<cmd>`, `<args>`, `<working-dir>`, and `<direct>` as follows:
```

Comment:
Sadly I don't think we can totally do away with the concept of `direct` in the platform spec until buildpack 0.8 has been formally deprecated. This section probably needs to branch on `direct` being true or false.

Also I don't know where exactly this would go, but the RFC points out
> user provided profiles will not be evaluated when using the new platform API, even when a shell process is executed
This seems like a relatively important point for users to understand. We might want to call it out explicitly.

Reply:
Agreed...this is pretty fundamental...going to mark as draft until we get this ironed out.

Reply:
Addressed this; ready for another review.